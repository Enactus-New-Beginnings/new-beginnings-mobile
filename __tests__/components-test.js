import React from 'react';
import renderer from 'react-test-renderer';
import Intro from '../screens/Home';

it('renders correctly', () => {
  const tree = renderer.create(<Intro />).toJSON();
  expect(tree).toMatchSnapshot();
});