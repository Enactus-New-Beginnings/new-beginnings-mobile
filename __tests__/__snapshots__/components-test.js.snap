// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "orange",
      "flex": 1,
    }
  }
>
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Object {
        "flex": 1,
        "justifyContent": "flex-end",
      }
    }
  >
    <Image
      resizeMode="cover"
      source={
        Object {
          "uri": "https://i.imgur.com/22nmNA1.png",
        }
      }
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": undefined,
            "width": undefined,
          },
          undefined,
        ]
      }
    />
    <View
      style={
        Object {
          "paddingTop": "10%",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#ff8575",
              "borderRadius": 10,
              "marginBottom": "5%",
              "marginLeft": "5%",
              "marginRight": "5%",
              "marginTop": "5%",
              "padding": 20,
            }
          }
        >
          <Text
            style={
              Object {
                "fontSize": 20,
                "fontWeight": "bold",
                "textAlign": "center",
              }
            }
          >
            Welcome to New Beginnings!
          </Text>
          <Text>
            Sign in or sign up for an account below to access personalized resources and mentorship programs near your location. 
          </Text>
        </View>
        <TextInput
          allowFontScaling={true}
          onChangeText={[Function]}
          placeholder="Email"
          rejectResponderTermination={true}
          style={
            Object {
              "backgroundColor": "white",
              "borderColor": "gray",
              "borderRadius": 10,
              "borderWidth": 1,
              "marginTop": 20,
              "padding": 10,
              "width": 600,
            }
          }
          underlineColorAndroid="transparent"
          value=""
        />
        <TextInput
          allowFontScaling={true}
          onChangeText={[Function]}
          placeholder="Password"
          rejectResponderTermination={true}
          secureTextEntry={true}
          style={
            Object {
              "backgroundColor": "white",
              "borderColor": "gray",
              "borderRadius": 10,
              "borderWidth": 1,
              "marginTop": 20,
              "padding": 10,
              "width": 600,
            }
          }
          underlineColorAndroid="transparent"
          value=""
        />
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "blue",
              "borderRadius": 4,
              "elevation": 3,
              "justifyContent": "center",
              "marginTop": "5%",
              "opacity": 1,
              "paddingHorizontal": 32,
              "paddingVertical": 12,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
              }
            }
          >
            Log In
          </Text>
        </View>
        <Text
          style={
            Object {
              "marginLeft": "5%",
              "marginRight": "5%",
              "marginTop": "5%",
            }
          }
        >
          Don't have an account? 
          <Text
            onPress={[Function]}
            style={
              Object {
                "color": "blue",
              }
            }
          >
            Sign Up
          </Text>
        </Text>
      </View>
    </View>
  </View>
</View>
`;
